packages/core/framework/src/http/express-loader.ts

```diff
@@ -72,21 +72,53 @@ export async function expressLoader({ app }: { app: Express }): Promise<{
     )
   }
 
+  let loggingMiddleware: RequestHandler
+
   /**
    * The middleware to use for logging. We write the log messages
    * using winston, but rely on morgan to hook into HTTP requests
    */
-  const loggingMiddleware = morgan(
-    IS_DEV
-      ? ":method :url ← :referrer (:status) - :response-time ms"
-      : "combined",
-    {
-      skip: shouldSkipHttpLog,
-      stream: {
-        write: (message: string) => logger.http(message.trim()),
-      },
+  if (!IS_DEV) {
+    const jsonFormat = (tokens, req, res) => {
+      const result = {
+        level: "http",
```

Comment:
should we call this `component` or something along the lines instead? `level` may be interpreted as `log_level`.

And could we add the log level as well? `info` in this case

Reply:
the log level was currently set to HTTP, which when we check was a valid level. Do you want to change it to info is that correct?

Reply:
<img width="875" alt="Screenshot 2025-02-17 at 13 31 35" src="https://github.com/user-attachments/assets/c45d449d-ead6-402c-a053-0b77c381cbb6" />

This log level was comming from winston which was following npm approach, let if you want it to be different and in that case I can switch to something else

Reply:
ah, didn't know `http` was an actual log level. I guess we can keep it then